qa/workunits/rbd/krbd_export_import.sh

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env bash
+set -eEx
+
+# This test performs the following steps:
+#
+# * creates an image file
+# * imports it and then maps it
+# * imports the mapped device and validates the content
+# * create a new blank image, maps it and then exports the image to it,
+#   validating its content.
+# * checks unsupported operations:
+#   * v2 exports/imports to/from raw block devices
+#   * diff exports/imports to/from raw block devices
```

Comment:
This bit is outdated now, but I'm anchoring a wider comment here:
- Using another RBD image as an export target/import source is confusing and krbd doesn't need to be dragged in.  I'd use loop device(s) based on regular files instead.
- We already have `qa/workunits/rbd/import_export.sh`, I'd rather see it extended than an additional workunit being added.

As an example, I'm thinking of something like:
```
# try again, importing from a block device
LOOP_DEV=$(sudo losetup -f --show ${TMPDIR}/img)
rbd import $RBD_CREATE_ARGS ${LOOP_DEV} testimg
sudo losetup -d ${LOOP_DEV}
rbd export testimg ${TMPDIR}/img2
rbd export testimg - > ${TMPDIR}/img3
rbd rm testimg
cmp ${TMPDIR}/img ${TMPDIR}/img2
cmp ${TMPDIR}/img ${TMPDIR}/img3
rm ${TMPDIR}/img2 ${TMPDIR}/img3
```
near `try again, importing from stdin` test there.  A similar approach can be followed for other tests.

Reply:
Indeed, I went with krbd for convenience but I'll switch to loopback devices.